tsdef/RemoteTrackPublication.d.ts

```diff
@@ -0,0 +1,19 @@
+import { Track } from './Track';
```

Comment:
Please export
```ts
export type RemoteTrack = RemoteAudioTrack | RemoteDataTrack | RemoteVideoTrack;
```
from a new file called `types.d.ts` and import it here:
```suggestion
import { Track } from './Track';
import { RemoteTrack } from './types';
```
Without this, references to `RemoteTrack` would point to an undefined type.